src/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -1028,6 +1028,23 @@ public IEqualityComparer<T> Comparer
             }
         }
 
+        /// <summary>
+        /// Ensures that the hashset can hold up to 'capacity' entries without any further expansion of its backing storage
```

Comment:
Nit: `hashset` should be either `hash set` or `HashSet` or `HashSet{T}`.  The sentence should also end with a period.